.gitignore

```diff
@@ -1,27 +1,106 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
```

Comment:
These files are generated by python 2, which dulwich no longer supports.

Reply:
This is Cython for the C language, and it is also your own original ignore list. I just moved the position. Many of them are originally there, but you said you want to delete.

I just merged the three into one

Reply:
And this is the most basic ignore list, this I would strongly recommend keeping it.